mrtt-ui/src/components/RestorationAimsForm/RestorationAimsForm.js

```diff
@@ -34,21 +59,21 @@ const RestorationAimsForm = () => {
   return (
     <MainFormDiv>
       <SectionFormTitle>Restoration Aims</SectionFormTitle>
-      <form onSubmit={validateInputs(handleSubmit)}>
+      <Form onSubmit={validateInputs(handleSubmit)}>
         <CheckboxGroupMangroveWithLabel
-          fieldName='ecologicalAims'
+          fieldName='1'
```

Comment:
react hook form wouldnt let me use a fieldName of `'Q3.1` It converted it to `'Q3'`.

Reply:
@julia-rom , Im assigning a fieldName to everything after the decimal place for a `question_id`. I dont love it, but its a workaround to the decimals getting lopped off even though they are strings. I could also do `fieldName='3_1` and then do a string replace. 

Either way, I think there needs to be a comment explaining the hack/weirdness